client/post-editor/editor-drawer/index.jsx

```diff
@@ -326,7 +327,10 @@ const EditorDrawer = React.createClass( {
 				{ site && ! this.hasHardCodedPostTypeSupports( type ) && (
 					<QueryPostTypes siteId={ site.ID } />
 				) }
-				{ this.renderTaxonomies() }
+				{ config.isEnabled( 'manage/custom-post-types' ) &&	! includes( [ 'post', 'page' ], this.props.type )
```

Comment:
Tab in here. In fact, I think we should drop the `! includes` part of the condition altogether, but fine to leave that as a separate task if you'd prefer.